posts/2023/mfdataset.ipynb

```diff
@@ -0,0 +1,8065 @@
+{
```

Comment:
<p>This is very helpful context that I think many people will find useful!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NCAR/esds/pull/238/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>